README.md

```diff
@@ -128,6 +128,19 @@ sudo pip3 install -r requirements.txt
 python3 start_qrl.py
 ```
 
+If you get the following error:
+
+```dependencies not satisfied, run [pip3 install -r requirements.txt] first.
+ The 'cffi>=1.7' distribution was not found and is required by cryptography
+ get an error with 
+``` 
+
+Add the cffi package:
+
+```
+[sudo] pip3 install cffi
```

Comment:
If this is the case, it would make more sense to add cffi to the requirements.txt file

Reply:
I made this suggestion to Cyyber too.
It look like it is not required on newer Linux version.